File: app/src/main/java/com/trilema/doctrinalang/ui/settings/SettingsFragment.java
package com.trilema.doctrinalang.ui.settings;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.trilema.doctrinalang.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey);
    }
}

File: app/src/main/java/com/trilema/doctrinalang/ui/settings/SettingsFragment.kt
package com.trilema.doctrinalang.ui.settings

import androidx.preference.PreferenceFragmentCompat
import android.os.Bundle
import com.trilema.doctrinalang.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle, rootKey: String) {
        setPreferencesFromResource(R.xml.root_preferences, rootKey)
    }
}